packages/core/src/components/badge/badge.scss

```diff
@@ -0,0 +1,28 @@
+:host {
+  display: inline-block;
+}
+
+.atom-badge {
+  border-radius: var(--spacing-base);
+  color: var(--color-neutral-white);
+  font-size: var(--text-badge);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Because this variable is a font, we must use `font` and use `letter-spacing`.
```suggestion
font: var(--text-badge);
letter-spacing: var(--text-badge-letter);
```

Reply:
ignore letter-spacing, I hadn't seen it. :hehe:

Reply:
Thanks!!

Made: https://github.com/juntossomosmais/atomium/pull/239/commits/7569153e55e3973cfc20ce95667b91cdaa2e5318